code/model/modelread.cpp

```diff
@@ -5655,7 +5655,7 @@ void parse_glowpoint_table(const char *filename)
 		}
 		required_string("#End");
 	} catch (const parse::ParseException& e) {
-		mprintf(("Unable to parse '%s'!  Error code = %d.\n", filename, e.what()));
+		mprintf(("Unable to parse '%s'!  Error code = %s.\n", filename, e.what()));
```

Comment:
In accordance with every other instance of `catch (const parse::ParseException& e)`, this should be changed to say "Error message" instead of "Error code".

...Actually, it looks like there's another case of "Error code" in wmcgui.cpp, but it already uses "%s" (or it presumably would've been covered in this diff).